lib/haml/temple_engine.rb

```diff
@@ -13,6 +13,7 @@ class TempleEngine < Temple::Engine
       encoding:             nil,
       escape_attrs:         true,
       escape_html:          false,
+      escape_interpolated_html: nil,
```

Comment:
To make sure it's a boolean flag, the default value should be `false` instead of `nil`.

Reply:
Ah, okay I understood you intention. So probably you intends to mean absence of this option here.